CMakeLists.txt

```diff
@@ -9,12 +9,68 @@ endif()
 
 project(Zeek C CXX)
 
+
+# We want to set ENABLE_DEBUG to ON by default if the build type is Debug.
+set(ENABLE_DEBUG_DEFAULT OFF)
+if ( NOT GENERATOR_IS_MULTI_CONFIG )
+  string(TOLOWER "${CMAKE_BUILD_TYPE}" build_type_lower)
+  if ( build_type_lower STREQUAL "debug" )
+    set(ENABLE_DEBUG_DEFAULT ON)
```

Comment:
Should this also be set if `RelWithDebInfo` is set? I can't remember all of what's different between those two modes.

Reply:
`RelWithDebInfo` should use similar flags to `Release`, except that the binary still contains debug symbols to allow you to jump into a debugger (or generate meaningful stack traces). The `configure` script enabled this flag only for actual debug builds and I think keeping it this way makes sense.